app/routes/forgot.js

```diff
@@ -0,0 +1,12 @@
+import Route from 'ember-route';
+
+export default Route.extend({
+  setupController() {
+    this.send('disableChrome');
```

Comment:
you need to call `this._super(...arguments)` here. but even better why not use the `didTransition` action?